### PR TITLE
Consistently abbreviate all journal names

### PR DESCRIPTION
**Pull request by michael-hartmann**
https://github.com/openjournals/joss-reviews/issues/2011

### CODE REVIEW

**Comment by gertingold** [comment]
I believe it should be the other way round.

**Comment by gertingold** [comment]
Ok, abbreviations don't seem to be a problem.

File: docs/paper/paper.bib [hunk 1]
```diff
@@ -147,7 +147,7 @@ @article{ambikasaran_joss_2019
   volume    = {4},
   number    = {34},
   pages     = {1167},
-  journal   = {The Journal of Open Source Software},
+  journal   = {J. Open Source Softw.},
```
Comment (gertingold) [inline docs/paper/paper.bib]: I wonder whether this should be JOSS?

Comment (michael-hartmann) [reply docs/paper/paper.bib]: This is JOSS. The abbrevation was suggested by Wikipedia.

Comment (gertingold) [reply docs/paper/paper.bib]: I just asked @danielskatz how to abbreviate this journal.

Comment (michael-hartmann) [reply docs/paper/paper.bib]: Good idea.

as a side note: the abbrevation "J. Open Source Softw." is according to the ISO 4 standard: https://paperpile.com/n/journal-of-open-source-software-abbreviation/

**Comment by gertingold** [comment]
I have merged, but we still need to decide whether "J. Open Source Softw." is fine.

**Pull request merged**